### ISSUE TITLE
Create an option/method for a user to set the location, timezone, etc

### ISSUE THREAD

**Issue by darianbjohnson** [Phambili-Tech/Newt_Display#5]
Right now, the Newt has limited ways to allow users to customize the display (without writing code). Need to think about a way to better support no-code user customizations. Maybe there is a way to do it with a website (versus handling it on-board).

**Comment by andypiper** [comment]
In principle it could be possible to use the WifiManager hooks to add additional variables and settings, although once you’re set up you probably don’t want to have to go back to the WifiManager, so something that could enable sending information via a website -> MQTT topic might be interesting. That said, that introduces a lot of security thoughts.

**Comment by darianbjohnson** [comment]
Yeah, exactly! So I need to think about this a bit. I kind of like how the remarkable tablet works. They have a public webpage; the user has to enter a 6 digit one time code [displayed on the device], which enables the user to make a few device settings. That might be an option... especially if I decide (at some point) to sync personal information [health data, calendars, etc].

**Comment by darianbjohnson** [comment]
Changing this to a "bug" - because the auto look-up incorrectly chooses the location for a few users (per issue created/commented by @kantarjiev and @hmeyer)

I'm considering three options:

1. Create a webpage where people can manually update settings, then request that the user do a "settings sync" (or, the sync happens automatically at the next wifi connection)
2. Have the Newt make a webpage available for a user to make changes. The user would need to force the Newt to serve as a wifiStation for this to work
3. Update the UI to support manually entry of a location. I've created a keyboard UI that can help with this, but it's not intuitive (the navigation is "klugey").

**Comment by hmeyer** [comment]
Personally, I'm strongly against 1)
That is, if we went for 1) the device would be (partially) broken, the moment that webpage is being shut down.

For 2) Why does it need to be a wifiStation? Couldn't it serve in client mode as well?
For 3) Could this be solved with a binary search like interface? In that case two buttons could be enough:
Left and Right.

Take the full list of options -> sort 

let the user choose between: list-start->list-mid  and list-mid->list-end
replace list be the selected sublist
repeat until the list contains only one element

**Comment by jondaddio** [comment]
Something has changed since I updated to 1.1.5 a while ago.  Before the update, time matched perfectly -- matched my GPS clock sitting on my desk.  After the update, I noticed the time was about 90 secs fast.  Today I did a full WiFi reset (multiple times) and I got 2.5 hours fast, and latest is ~65 minutes fast.

Didi something change?  The location and time zone are correct.
